app-backend/common/src/main/scala/S3.scala

```diff
@@ -0,0 +1,37 @@
+package com.azavea.rf.common
+
+import scala.util.{Success, Failure, Try}
+
+import org.apache.commons.io.IOUtils
+
+import com.amazonaws.auth._
+import com.amazonaws.services.s3.{AmazonS3Client, AmazonS3URI}
+import com.amazonaws.services.s3.model.{S3Object, ObjectMetadata}
+
+import java.io._
+import java.net._
+
+package object S3 {
```

Comment:
There is some similar functionality here which I think I'll need to take advantage of for some of the tiler URL refactoring, so it might be worth moving that out into the common project: https://github.com/azavea/raster-foundry/blob/develop/app-backend/ingest/src/main/scala/com/azavea/rf/ingest/util/S3.scala